Source/WebCore/editing/markup.cpp

```diff
@@ -104,8 +104,10 @@ class AttributeChange {
     {
     }
 
-    AttributeChange(Element* element, const QualifiedName& name, const AtomString& value)
-        : m_element(element), m_name(name), m_value(value)
+    AttributeChange(RefPtr<Element>&& element, const QualifiedName& name, const AtomString& value)
```

Comment:
We should move in the value instead of copying. Could change the type of the argument to `AtomString`. In fact, since this stores the name and value neither should be `const&`; never does any good.